projects/gis/src/app/app-routing.module.ts

```diff
@@ -1,23 +1,25 @@
 import {NgModule} from '@angular/core';
 import {RouterModule, Routes} from '@angular/router';
-import {IsLoggedInGuard} from '@geoengine/core';
+import {BackendStatusPageComponent, LogInGuard, NotFoundComponent, BackendAvailableGuard} from '@geoengine/core';
 import {CanRegisterGuard} from './guards/can-register.guard';
 import {LoginComponent} from './login/login.component';
 import {MainComponent} from './main/main.component';
 import {RegisterComponent} from './register/register.component';
 
 const routes: Routes = [
     {path: '', redirectTo: 'map', pathMatch: 'full'},
-    {path: 'map', component: MainComponent, canActivate: [IsLoggedInGuard]},
+    {path: 'map', component: MainComponent, canActivate: [BackendAvailableGuard, LogInGuard]},
     {path: 'signin', component: LoginComponent},
```

Comment:
You don't land here anymore when you set

```
anonymous_access = false
```

Reply:
it works for me